### ISSUE TITLE
Error: invalid byte sequence in US-ASCII

### ISSUE THREAD

**Issue by rdschaffer** [Homebrew/homebrew#45781]
I have not used brew for very long. I recently upgraded to El Capitan.

I have run brew update and upgrade successfully.

brew doctor gives:
```
Error: invalid byte sequence in US-ASCII
Please report this bug:
    https://git.io/brew-troubleshooting
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:45:in `=~'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:45:in `chop_basename'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:341:in `plus'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:334:in `+'
/usr/local/Library/Homebrew/extend/pathname.rb:312:in `resolved_path_exists?'
/usr/local/Library/Homebrew/cmd/doctor.rb:232:in `block (2 levels) in check_for_broken_symlinks'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:546:in `block in find'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/find.rb:43:in `block in find'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/find.rb:42:in `catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/find.rb:42:in `find'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:546:in `find'
/usr/local/Library/Homebrew/cmd/doctor.rb:231:in `block in check_for_broken_symlinks'
/usr/local/Library/Homebrew/cmd/doctor.rb:229:in `each'
/usr/local/Library/Homebrew/cmd/doctor.rb:229:in `check_for_broken_symlinks'
/usr/local/Library/Homebrew/cmd/doctor.rb:1314:in `block in doctor'
/usr/local/Library/Homebrew/cmd/doctor.rb:1312:in `each'
/usr/local/Library/Homebrew/cmd/doctor.rb:1312:in `doctor'
/usr/local/Library/brew.rb:146:in `<main>'
```
Not sure what I should do.

       thanks, RD Schaffer

**Comment by bfontaine** [comment]
Does `brew readall` work?

**Comment by rdschaffer** [comment]
It runs, but returns nothing.


> On 7 Nov, 2015, at 16:16, Baptiste Fontaine <notifications@github.com> wrote:
> 
> Does brew readall work?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Homebrew/homebrew/issues/45781#issuecomment-154713638>.
> 

Email:       R.D.Schaffer@cern.ch 
tel(Orsay): +33 1 64 46 83 78   tel(cern)   : +41 22 76 712 67  
Address:  LAL BAT 200         
                BP 34                      
                91898 ORSAY
                France

**Comment by DomT4** [comment]
El Cap probably reset your locale. Try `export LC_ALL=en_US.UTF-8` (Or whatever country code before the `.UTF-8` you prefer and then run the command again.

**Comment by rdschaffer** [comment]
Hi there,

  OK, ‘brew doctor’ works with this. Do I need to do this in my .zshrc?

   thanks, RD

> On 7 Nov, 2015, at 17:50, Dominyk Tiller <notifications@github.com> wrote:
> 
> El Cap probably reset your locale. Try export LC_ALL=en_US.UTF-8 (Or whatever country code before the .UTF-8 you prefer and then run the command again.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Homebrew/homebrew/issues/45781#issuecomment-154722941>.
> 

Email:       R.D.Schaffer@cern.ch 
tel(Orsay): +33 1 64 46 83 78   tel(cern)   : +41 22 76 712 67  
Address:  LAL BAT 200         
                BP 34                      
                91898 ORSAY
                France

**Comment by DomT4** [comment]
Yeah. It's something I've been meaning to fix for a while so it doesn't require user intervention.

**Comment by rdschaffer** [comment]
OK, thanks. RD

> On 7 Nov, 2015, at 18:25, Dominyk Tiller <notifications@github.com> wrote:
> 
> Yeah. It's something I've been meaning to fix for a while so it doesn't require user intervention.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Homebrew/homebrew/issues/45781#issuecomment-154725409>.
> 

Email:       R.D.Schaffer@cern.ch 
tel(Orsay): +33 1 64 46 83 78   tel(cern)   : +41 22 76 712 67  
Address:  LAL BAT 200         
                BP 34                      
                91898 ORSAY
                France

**Comment by DomT4** [comment]
Will close this out as resolved for you, but feel free to track #44744 if you'd like to. Tracking the wider bug in that thread.
